app/views/pages/home.html.erb

```diff
@@ -1,2 +1,18 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<section class="home">
+	<div class="container pt-5">
+		<div class="row">
+			<div class="col-sm-12 col-md-12 col-lg-6">
+				<div class="home_text pt-5">
+					<h1>Donate to a Lonley Soldier</h1>
```

Comment:
i think maybe it should say "Donate to a lone soldier" as opposed to "lonely soldier"

Reply:
changed!